### ISSUE TITLE
Temperature calculations for Hargreaves evaporation

### ISSUE THREAD

**Issue by MikeGregory-Canada** [USEPA/Stormwater-Management-Model#72]
I’m getting PET values that are about half of what I expect and I think it’s because SWMM5 is using the wrong temperature values. See attached files in “InternationalFallsMN_DailyRainEvap.zip”. The GHCN-Daily climate summary file “InternationalFallsMN_DailyGHCN_1907-2020.txt” has TMIN and TMAX values that range between -55 and 103 deg-F. However, when I plot the System Temperature, it shows a range between 24-50 deg-F (the average daily temp should be between -30 and 83 deg-F).
![image](https://user-images.githubusercontent.com/70409362/118716924-66872180-b7f3-11eb-96fc-782f7a6ff2e3.png)
[InternationalFallsMN_DailyRainEvap.zip](https://github.com/USEPA/Stormwater-Management-Model/files/6504184/InternationalFallsMN_DailyRainEvap.zip)

**Comment by LRossman** [comment]
SWMM assumes that GHCN-Daily data follow the format described in a [readme](https://www1.ncdc.noaa.gov/pub/data/ghcn/daily/readme.txt) file posted on NOAA's web site. This format has TMIN and TMAX expressed as tenths of degrees Celsius (e.g., a value of 223 would be 22.3 degrees C). I believe that this format also applied to data downloaded from NOAA's Climate Data Online (CDO) service at least up until 2015 or so, since that is how it's described in the SWMM Reference Manual Vol. I - Hydrology (see pages 40 - 41). However it now appears that CDO temperature data is downloaded as actual values (not tenths) that can be either degrees C or F as requested by the user. The climate file used by @MikeGregory-Canada is in this newer format and therefore SWMM is processing it incorrectly.

**Comment by MikeGregory-Canada** [comment]
OK - thanks for clarifying Lew!

**Comment by MikeGregory-Canada** [comment]
Indeed, converting TMAX and TMIN back to 10×Celsius degrees does the trick. Also, note that AWND is newly formatted from CDO (in decimal ft/s), it also needs to be converted back to units of 10×m/s. 
![image](https://user-images.githubusercontent.com/70409362/118868877-b8db4780-b8b2-11eb-97b4-134fd13db464.png)

**Comment by LRossman** [comment]
To avoid this issue occurring in the future I propose extending the `FILE `entry in the `[TEMPERATURE]` section of the SWMM input file as follows:
```
[TEMPERATURE]
FILE  FileName  (StartDate) (Units)
```
where the new `Units `item can be either `C10`, `C`, or `F` for degrees Centigrade times 10, degrees Centigrade, and degrees Farenheit, respectively. The code would be modified to use this item to properly interpret what units a GHCND file is using for its data. Sound good?

**Comment by MikeGregory-Canada** [comment]
Excellent idea! Thanks again Lew.

**Comment by michaeltryby** [comment]
Closed in Release v5.2.0
